### PR TITLE
Improve UI for org-card component when it displays distance

### PR DESCRIPTION
**Pull request by LouiaR**
=> Add unit: miles
=> Round distance to one decimal place

### CODE REVIEW

File: frontend/src/Components/Organisation/index.css [hunk 1]
```diff
@@ -49,9 +49,12 @@
...
-  color: #1abcd4;
-  border-bottom: .1em solid gray;
-  padding-bottom: .5em;
-  margin-bottom: .5em;
+.org-distance{
+  display: flex;
+  justify-content: space-between;
 }
+
+.org-distance { color: #666; }
```
Comment (kkarimi) [inline frontend/src/Components/Organisation/index.css]: @LouiaR can we just have one definition of class? right now there is two `.org-distance`

File: frontend/src/Components/Organisation/OrganisationCard.js [hunk 2]
```diff
@@ -32,8 +32,15 @@ const OrganisationCard = ({
...
-    {org.distance ? <h1 className="distance">{org.distance}</h1> : null}
-    { org.org_name ? <h1>{org.org_name}</h1> : <h1 className="not-available"> Add organisation name ...</h1>}
+    
+    { org.org_name ? 
+      <span className={org.distance ? 'org-distance': ''}>
+        <h1>{org.org_name}</h1> 
+        {org.distance ? <p className="">{(org.distance).toFixed(1)} miles</p> : null}
+      </span>
+      : 
+      <h1 className="not-available"> Add organisation name ...</h1>
```
Comment (kkarimi) [inline frontend/src/Components/Organisation/OrganisationCard.js]: Are there many orgs with no name?

Comment (LouiaR) [reply frontend/src/Components/Organisation/OrganisationCard.js]: I haven't checked all the orgs  inside the new db, I was mean to handle org that didn't have name on the previous db, I will check later and If there is no org I will remove it.

File: frontend/src/Components/Organisation/index.css (hunk 1 above)
Comment (LouiaR) [reply frontend/src/Components/Organisation/index.css]: Good spot my mistake

**Pull request merged**